managers/templates/managers/project_task_activities.html

```diff
@@ -0,0 +1,74 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
+{% load static %}
+
+{% block extra_head %}
+    <link rel="stylesheet" type="text/css" href="{% static 'managers/style.css' %}"/>
+{% endblock %}
+
+{% block content %}
+    <h1>
+        <small>
+            <a href="{% url 'custom-project-detail' pk=project.pk %}">
+                <span class="glyphicon glyphicon-chevron-left"></span>
+            </a>
+        </small>
+        {% trans 'Project task activities' %}
+    </h1>
+    <div class="container">
+        <h5>{% trans 'Add new task activity or choose from existing one' %}</h5>
+        <form method="post" action="." class="add-task-activity">
+            {% csrf_token %}
+            {% trans 'Name:' %}
+            {{ form.name }}
+            <button class="add-task-activity-button" type="submit" title="{% trans 'Add new task activity' %}"><span
+                    class="glyphicon glyphicon-plus"></span></button>
+        </form>
+        <p>
+            {% if activities_not_connected %}
+                <form method="get" action=".">
+                    <select class="multiselect-task-activities" multiple="multiple" name="task_activities" size="6"
+                            title="{% trans 'If you want add more than one press ctrl' %}">
+                        {% for task in activities_not_connected %}
+                            <option value="{{ task.id }}">{{ task.name }}</option>
+                        {% endfor %}
+                    </select>
+                <p>
+                    <input type="submit" class="btn btn-primary" value="Add to project">
+                </p>
+                </form>
+                </p>
+            {% endif %}
+    </div>
+    <ul>
+        <div class="container">
+            <div class="table-responsive">
+                <table class="table">
+                    <tr>
+                        <th class="task-activities-column">{% trans 'Index' %}</th>
+                        <th class="task-activities-column">{% trans 'Name' %}</th>
+                        <th class="task-activities-column">{% trans 'Default' %}</th>
+                        <td class=Invisible></td>
+                    </tr>
+                    {% for task_activity in task_activities %}
+                        <tr>
+                            <td class="task-activities-column">{{ forloop.counter }}</td>
+                            <td class="task-activities-column">{{ task_activity.name }}</td>
+                            <td class="task-activities-column">{{ task_activity.is_default }}</td>
+                            <td class=Invisible align="center">
+                                <a href=".?task_activity={{ task_activity.id }}"
```

Comment:
Don't do this, use `{% url 'xxx' %}?task_activity={{ task_activity.id }}` instead of `.`.